### PR TITLE
add cross account CF stack files

### LINKED ISSUE #5: Grant AWS IAM access to devs
- [x] Decision on model for delegated access to parsys.  Recommendation on scoping.
- [x] Update IAM managed policy granting access
- [ ] Deliver document to tasos and nemo with instructions on deploying the apex project as a developer.

### PR DESCRIPTION
**Pull request by andrewkrug**
Closes #5 

Tested.  Need to do a training session on how to use these with APEX but it meets the story card.

### CODE REVIEW

File: cloudformation/PolicyTemplates/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## CIS Developer Policy Templates
```
Comment (johngian) [inline cloudformation/PolicyTemplates/README.md]: I suggest we move this to docs instead.

File: cloudformation/PolicyTemplates/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    pip install -r requirements.txt
+    Edit your ~/.aws/credentials file and create a profile that looks like this with static access keys that give you the ability to assume_role.
+
+[defult-long-term]
+aws_access_key_id = YOUR_LONGTERM_KEY_ID
+aws_secret_access_key = YOUR_LONGTERM_ACCESS_KEY
+
+I set up a bash alias for convenience after this that looks like:
+
+alias cisaws="/Users/akrug/Library/Python/2.7/bin/aws-mfa --device {YOUR_MFA_ARN} --assume-role arn:aws:iam::656532927350:role/CISDeveloper --role-session-name "cis-developer-session""
```
Comment (johngian) [inline cloudformation/PolicyTemplates/README.md]: Let's avoid adding user paths in the docs. Just ``alias cisaws=aws-mfa .....`` should be good.

File: cloudformation/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+#
```
Comment (johngian) [inline cloudformation/README.md]: Please remove empty file

File: cloudformation/cis-streams-and-vault.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        Value: ChangeIntegration
+      - Key: Type
+        Value: Input
+  ### DynamoDB Table of all users
+  IdentityVaultUsers:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+      -
+        AttributeName: "UserId"
```
Comment (johngian) [inline cloudformation/cis-streams-and-vault.yml]: I think that this one should be ``user_id`` after https://github.com/mozilla-iam/cis/pull/8/files#diff-16da26b86d8a5654e680f1d3757537f4R6

File: cloudformation/cis-streams-and-vault.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        AttributeType: "S"
+      KeySchema:
+      -
+        AttributeName: "UserId"
```
Comment (johngian) [inline cloudformation/cis-streams-and-vault.yml]: Same here.

File: cloudformation/cis-streams-and-vault.yml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        WriteCapacityUnits: "1"
+      StreamSpecification:
+        StreamViewType: NEW_IMAGE
+  ### Groups table populated by loopback stream driver for fast search of group members
+  IdentityVaultGroups:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+      -
+        AttributeName: "GroupId"
```
Comment (johngian) [inline cloudformation/cis-streams-and-vault.yml]: Same here.

File: cloudformation/cis-streams-and-vault.yml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        AttributeType: "S"
+      KeySchema:
+      -
+        AttributeName: "GroupId"
```
Comment (johngian) [inline cloudformation/cis-streams-and-vault.yml]: Same here.

**Comment by johngian** [comment]
Please remove ``.DS_Store``

File: cloudformation/PolicyTemplates/README.md (hunk 1 above)
Comment (andrewkrug) [inline cloudformation/PolicyTemplates/README.md]: Is it cool to do this later?  I agree should be in read the docs ultimately.  Should this block merging?

Comment (johngian) [inline cloudformation/PolicyTemplates/README.md]: Yes let's do this later. Please add an issue for that and let's move on.

**Pull request closed without merging**